src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js

```diff
@@ -156,7 +158,10 @@ export default class geFormWidgetTokenizeCard extends LightningElement {
                     if (message.error) {
                         reject(this.handleTokenizationError(message));
                     } else if (message.token) {
-                        resolve({ [DATA_IMPORT_PAYMENT_AUTHORIZATION_TOKEN_FIELD.fieldApiName]: message.token });
+                        resolve({
+                            [DATA_IMPORT_PAYMENT_AUTHORIZATION_TOKEN_FIELD.fieldApiName]: message.token,
+                            [DATA_IMPORT_PAYMENT_STATUS_FIELD.fieldApiName]: PAYMENT_TRANSACTION_STATUS_ENUM.PENDING
```

Comment:
This is awesome once we align with however way we're retrieving the transaction status enums.

Reply:
We're aligned here now, pulling the pending value from the back-end.